src/plugins/so_management/server/lib/find_all.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { SavedObjectsClientContract, SavedObject, SavedObjectsFindOptions } from 'src/core/server';
+
+export const findAll = async (
+  client: SavedObjectsClientContract,
+  findOptions: SavedObjectsFindOptions
+): Promise<SavedObject[]> => {
+  return recursiveFind(client, findOptions, 1, []);
+};
+
+const recursiveFind = async (
```

Comment:
I added test for them, but appart from that the `so_management/server/lib` utilities were almost migrated to TS without any changes